Assets/HoloToolkit/Utilities/Editor/AutoConfigureMenu.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.IO;
+using System.Text;
+using System.Text.RegularExpressions;
+using UnityEditor;
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// Configuration options derived from here: 
+    /// https://developer.microsoft.com/en-us/windows/holographic/unity_development_overview#Configuring_a_Unity_project_for_HoloLens
+    /// </summary>
+
+    
+    public class AutoConfigureMenu : MonoBehaviour
+    {
+
+        [MenuItem("HoloToolkit/Configure/Show Help", false, 3)]
+        public static void ShowHelp()
+        {
+            Application.OpenURL("https://developer.microsoft.com/en-us/windows/holographic/unity_development_overview#Configuring_a_Unity_project_for_HoloLens");
```

Comment:
I don't think GettingStarted.md makes sense because that mostly includes instructions on how to import the toolkit (which will already be done by the time this menu item is available). I'm going to point it to the wiki instead. This can be changed if someone feels strongly about using a different URL.